.gitignore

```diff
@@ -1 +1,3 @@
 target
+Cargo.lock
```

Comment:
We are building an application, not a library. Much as with Rails, we want the lock checked in in this circumstance.